.github/workflows/ci.yml

```
@@ -0,0 +1,18 @@
+name: CI
+
+on:
+  push:
+    branches: ["master"]
+  pull_request:
+    branches: ["master"]
```

We'll do a branch rename soon

```suggestion
    branches: ["master", "main"]
```